packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer" ...attributes>
+  {{!-- {{yield to="topContent"}} --}}
+  {{yield (hash ExtraContent=(component "hds/app-footer/extra-content"))}}
+
+  <ul class="hds-app-footer__list">
+    {{yield
+      (hash
+        Item=(component "hds/app-footer/item")
+        Link=(component "hds/app-footer/link")
+        StatusLink=(component "hds/app-footer/status-link")
+        LegalLinks=(component "hds/app-footer/legal-links")
+      )
+    }}
+  </ul>
```

Comment:
from their schema (see above) seems they need another generic container after the list, but before the copyright, similar to the one you have above (but it's not the `bottomContent`, it flows with the rest of the items).